change/@memberjunction-core-4de67c01-619b-43b5-b9e8-0765f083cc62.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Applying package updates [skip ci]",
+  "packageName": "@memberjunction/core",
+  "email": "97354817+AN-BC@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

```suggestion
  "dependentChangeType": "minor"
```

For some reason, Beachball won't bump the minor version unless this is also `minor` but doesn't set it when you select `minor` in the CLI. This is one of the reasons I plan to move to using [changesets](https://github.com/changesets/changesets) at some point